plutus-playground-client/test/JsonEncodingTests.purs

```diff
@@ -145,9 +148,16 @@ arbitraryAssocMap genK genV = do
 
 arbitraryValue :: Gen Value
 arbitraryValue = do
-  assocMap <- arbitraryAssocMap arbitraryCurrencySymbol (arbitraryAssocMap arbitraryTokenName arbitrary)
+  assocMap <- arbitraryAssocMap arbitraryCurrencySymbol (arbitraryAssocMap arbitraryTokenName arbitraryBigInteger)
   pure $ Value { getValue: assocMap }
 
+arbitraryBigInteger :: Gen BigInteger
+arbitraryBigInteger = do
+  let
+    intSized :: Gen BigInteger
+    intSized = BigInteger.fromInt <$> arbitrary
+  product <$> vectorOf 5 intSized
```

Comment:
does this produce big big integers?

Reply:
Yes. Really nice big ones. Huge.

Reply:
ho though, isn't it limited by `Int` since it uses `BigInteger.formInt`?

Reply:
Yeah, so what we do is get a vector of 5 ints and multiply them. I've checked it and it produces a nice wide range, positive and negative.